applications/salk-portal/src/pages/ExperimentsPage.tsx

```diff
@@ -22,25 +31,34 @@ const useStyles = makeStyles({
         padding: '0.5rem',
 
         '& *': {
-          fontFamily: font
+            fontFamily: font
         },
 
         '&> div': {
-          height: '100%',
-          '&> div': {
-            position: 'relative',
-          }
+            height: '100%',
+            '&> div': {
+                position: 'relative',
+            }
         }
     }
 });
 
-export const CanvasWidget = {
-    id: 'canvasWidget',
-    name: "Spinal Cord Atlas",
-    component: "experimentViewer",
-    panelName: "leftPanel",
-    enableClose: false,
-    status: WidgetStatus.ACTIVE,
+const MOCKED_ID = 1
+
+export const CanvasWidget = (selectedAtlas: AtlasChoice, activeSubdivisions: Set<string>, activePopulations: any) => {
```

Comment:
@afonsobspinto any reason why this is in this file and not in a separate file (e.g. widgets/CanvasWidget.js)?